tests/lib/constants/constants.py

```diff
@@ -16,8 +16,13 @@
 
 INVALID_ROUTING_PLAN_DEV = "ae0f772e-6660-4829-8f11-1ed8a3fc68c2"
 
+DUPLICATE_ROUTING_PLAN_TEMPLATE_ID_INT = "bb454d66-033b-45c0-bbb6-d9b3420a0bd4"
 DUPLICATE_ROUTING_PLAN_TEMPLATE_ID_DEV = "bb454d66-033b-45c0-bbb6-d9b3420a0bd4"
 
+MISSING_TEMPLATE_ROUTING_PLANS_INT = [
```

Comment:
Do we need these to be separate as they're the same as internal-dev? same with duplicate routing plan template

Reply:
I've made them all environment agnostic.